src/Forms/HTMLEditor/TinyMCEConfig.php

```diff
@@ -593,7 +593,7 @@ public function insertButtonsAfter($after, $buttons)
 
     /**
      * Remove the first occurrence of buttons
-     * @param string ...$buttons one or more strings - the name of the buttons to remove
+     * @param string|array $buttons one or more strings - the name of the buttons to remove
```

Comment:
there are actually two signatures to this method.

1. Multiple string arguments
2. an array of strings

The `...$buttons` bit indicates the fact that all the string arguments are gathered into the `$buttons` argument, so this change looks like it will remove one valid docblock definition for another.

I think the correct definition here is: `@param string|string[] $buttons,... An array of strings or one or more string arguments - the name of the buttons to remove`

see https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.param.pkg.html